test/e2e/specs/editor/various/multi-block-selection.spec.js

```diff
@@ -693,6 +693,16 @@ test.describe( 'Multi-block selection', () => {
 			force: true,
 		} );
 
+		await expect
+			.poll( multiBlockSelectionUtils.getSelectedFlatIndices )
+			.toEqual( [ 1 ] );
+
+		await paragraph1.click( {
+			position: { x: -1, y: 0 },
+			// Use force since it's outside the bounding box of the element.
+			force: true,
+		} );
```

Comment:
Note that I am merely making things consistent here with the Esc shortcut: 

https://github.com/WordPress/gutenberg/blob/395b18f1c6850fc43271edd2d0114bdf79ed7bd7/packages/block-editor/src/components/block-tools/index.js#L163-L170